src/App.jsx

```diff
@@ -1,29 +1,39 @@
-import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import './App.css';
-import Header from './Header';
-import Questions from './Questions';
-import Answer from './Answer';
-import QuestionData from './Questions.json';
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <div className="Content">
-          <Header />
-          <h2>Manager is ready to help you!</h2>
-          <Route exact path="/" component={Questions} />
-          <Route
-            path="/question/:questionId"
-            render={({ match }) => (<Answer data={QuestionData.find(item =>
-            item.id.toString() === match.params.questionId)}
-            />
-       )} />
-        </div>
-      </Router>
-    );
-  }
-}
-
-export default App;
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import './App.css';
+import Header from './Header';
+import Questions from './Questions';
+import Answer from './Answer';
+import QuestionData from './Questions.json';
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <div>
+          <Header />
+          <h2>Manager is ready to help you!</h2>
+          <Route
+            exact
+            path="/"
+            render={() => (
+              <Questions list={QuestionData} />
+            )}
+          />
+          <Route
+            path="/question/:questionId"
+            render={({ match }) => (
+              <Answer
+                data={QuestionData.find(item => (
+                  item.id.toString() === match.params.questionId
```

Comment:
Bring out scope of the render method (the same for the function above) so that when the component is re-rendered, the function would not re-created every time